app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -27,10 +27,9 @@ def perform(form526_submission_id, full_name)
         send_fast_track_engineer_email_for_testing(form526_submission_id)
 
         bp_readings = FastTrack::HypertensionObservationData.new(client.get_resource('observations')).transform
-        return if no_recent_bp_readings(bp_readings)
```

Comment:
We can remove the `no_recent_bp_readings` method now!

Reply:
Updated in 6816760